components/src/styles/_mixins.scss

```diff
@@ -82,8 +82,7 @@
   $width: map-get($map, width) or nth-value($arglist, 1);
   $ratio: map-get($map, ratio) or
     if($width and $height, $width/$height, nth-value($arglist, 1)) or 1;
-  // $padding: 1 / $ratio * 100%; // this returns a warning
-  $padding: calc(1 / $ratio * 100%); // this is suggested by the sass compiler
+  $padding: calc((1 / #{$ratio}) * 100%);
```

Comment:
What does this line do?

Reply:
This mixin we use in calendar component, this gives us the ability to have correct aspect ratio for boxes(days) in calendar.
dart saas compiler has deprecated the direct use of division operations, now need to either import math module or use css's native calc method. https://sass-lang.com/documentation/breaking-changes/slash-div

This line will calculate the padding according to the aspect ratio provided to the mixin. `#{$ratio}` will be replaced by sass compiler. (previously it wasn't)